### ISSUE TITLE
cannot pass `make lint` after `cargo update`

### ISSUE THREAD

**Issue by ClSlaid** [datafuselabs/databend#7808]
### Search before asking

- [X] I had searched in the [issues](https://github.com/datafuselabs/databend/issues) and found no similar issues.

### Version

[d747a6c](https://github.com/datafuselabs/databend/commit/d747a6c5fd1e359fd0c1a345415c07f23c95db08)

### What's Wrong?

After `cargo update` all dependency cache, `make lint` will fail on checking `common-hive-meta-store`:

```
error[E0432]: unresolved import `thrift::protocol::TSerializable`
  --> /databend/target/debug/build/common-hive-meta-store-a3775ca046a58940/out/hms_patched.rs:16:142
   |
16 | ...ssageType, TInputProtocol, TOutputProtocol, TSerializable, TSetIdentifier, TStructIdentifier, TType};
   |                                                ^^^^^^^^^^^^^ no `TSerializable` in `protocol`
```

### How to Reproduce?

check out the latest main branch like [d747a6c](https://github.com/datafuselabs/databend/commit/d747a6c5fd1e359fd0c1a345415c07f23c95db08), and run `cargo update`, then `make lint`.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by Xuanwo** [comment]
Thrift released v0.17 serval days ago, but its rust lib is not updated. If you are using Archlinux, please rollback to v0.16 until the upstream fixed this problem.

**Comment by Xuanwo** [comment]
Already fixed.
